test/lib/sun/hotspot/WhiteBox.java

```diff
@@ -310,7 +310,7 @@ public         void    makeMethodNotCompilable(Executable method, int compLevel,
     makeMethodNotCompilable0(method, compLevel, isOsr);
   }
   public        int     getMethodCompilationLevel(Executable method) {
-    return getMethodCompilationLevel(method, false /*not ost*/);
+    return getMethodCompilationLevel(method, false /*not osr*/);
```

Comment:
This change is unrelated, should be fixed in a separate CR?

Reply:
This is a small typo fix, can I merge it in this patch?

Reply:
I think the convention is "no"

Reply:
Got it. Thanks for your review. I have removed this change.